spec/project/project_spec.rb

```diff
@@ -166,11 +166,13 @@
 
   describe '#set_user_roles' do
     it 'Properly updates user roles as needed' do
-      users_to_import = @domain.users.drop(rand(100)).take(5).map { |u| { user: u, role: 'admin' } }
+      users =  ProjectHelper.ensure_users(client: @client, amount: 5, caller: $example_name)
```

Comment:
Style/SpaceAroundOperators: Operator `=` should be surrounded by a single space.